bazel/google_cloud_cpp_deps.bzl

```diff
@@ -66,7 +121,8 @@ def google_cloud_cpp_deps(name = None):
         strip_prefix = "abseil-cpp-20230125.3",
     )
 
-    # Load a version of googletest that we know works.
+    # Load a version of googletest that we know works. This is needed to create
+    # google_cloud_cpp_*_mocks libraries.
```

Comment:
nit: `//:*_mocks` instead of `google_cloud_cpp_*_mocks` ?

(and we just pretend that `//:mocks` matches the regex)